modules/learn/pages/clusters-and-availability/automatic-failover.adoc

```diff
@@ -206,10 +248,7 @@ The value is incremented by 1 for every node that has an automatic-failover that
 * _Enablement of disk-related automatic failover; with corresponding time-period_.
```

Comment:
This comment is in reference to line 247 which describes the "Count".
It should say:

The value is incremented by 1 for every node that has an automatic-failover that occurs, up to the defined maximum count: beyond this point, no further automatic failover can be triggered until the count is reset to 0.  Running a rebalance will reset the count value back to 0.

cc @BenHuddleston